AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-non-availability-table/vho-work-hours-non-availability-table.component.ts

```diff
@@ -480,4 +480,7 @@ export class VhoWorkHoursNonAvailabilityTableComponent implements OnInit, CanDea
     resetStartDateAndEndDate() {
         this.filterForm.setValue({ startDate: null, endDate: null });
     }
+    get checkVhoHasWorkHours(): boolean {
```

Comment:
More of a naming style here but not a deal breaker. If returning a bool, prefix the method with 'if' or 'has'. turns this into a question to make it more readable and reflective of the behaviour. for example. `hasVHOGotWorkingHours`

Reply:
Thank you Shead .